.azuredevops/modulePipelines/ms.databricks.workspaces.yml

```diff
@@ -30,6 +30,7 @@ trigger:
       - '/.azuredevops/modulePipelines/ms.databricks.workspaces.yml'
       - '/.azuredevops/pipelineTemplates/*.yml'
       - '/modules/databricks/workspaces/*'
+      - '/modules/network/private-endpoints/*'
```

Comment:
Could we please hold on this until singular renames? 🙏

Reply:
I started working on this before starting on the singular names. We can definitely hold on to this - I will convert this PR to singular manually after we merged the other PRs.

Reply:
Blocking comment until singular names PR (#3573) is merged